metrics/metrics.go

```diff
@@ -15,97 +15,109 @@ import (
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
+const metricsNamespace = "certmgr"
+
 var (
 	startTime time.Time
 
-	// WatchCount counts the number of certificates being watched.
-	WatchCount = prometheus.NewCounterVec(
-		prometheus.CounterOpts{
-			Name: "cert_watching",
-			Help: "Number of certificates being watched",
+	// SpecWatchCount counts the number of specs being watched.
+	SpecWatchCount = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: metricsNamespace,
+			Name:      "specs_watched_total",
+			Help:      "Number of specs being watched",
 		},
-		[]string{"spec_path", "svcmgr", "cert_action", "cert_age", "ca", "ca_age"},
+		[]string{"spec_path", "svcmgr", "action", "ca"},
```

Comment:
Just to confirm: we don't want to monitor CA age? 
My understanding was we were interesting in doing this in case there's any CA cert inconsistencies.

Reply:
We do want to monitor CA (and cert) age- trace the Expires metric usage.  I've moved the age from a label value into a raw metric value we now export.

Reply:
The change I made results in metrics like thus:
```
# HELP certmgr_cert_expire_timestamp The unix time for when the given spec and type expires
# TYPE certmgr_cert_expire_timestamp gauge
certmgr_cert_expire_timestamp{spec_path="/etc/certmgr.d/etcd.json",type="ca"} 1.76099424e+09
certmgr_cert_expire_timestamp{spec_path="/etc/certmgr.d/etcd.json",type="cert"} 1.5762129e+09
certmgr_cert_expire_timestamp{spec_path="/etc/certmgr.d/kubernetes.json",type="ca"} 1.76099424e+09
certmgr_cert_expire_timestamp{spec_path="/etc/certmgr.d/kubernetes.json",type="cert"} 1.5761352e+09
```

any questions/concerns, please reopen the conversation.